features/support/server.js

```diff
@@ -0,0 +1,8 @@
+require('http')
+  .createServer((req, res) => {
+    if (req.url === '/message') {
+      res.writeHead(200, { 'Content-Type': 'text/plain' });
+      res.end('Hello World!\n');
+    }
```

Comment:
Maybe it would be nicer to handle other paths even if it is just 500ing, otherwise the server will just hang unless you do /message request.

```suggestion
    } else {
      res.writeHead(500);
      res.end();
    }
```

Reply:
Yeah, I'm a bit torn between having the server as brief as possible (because it's inlined in the feature files) and having it complete (because that could make debugging easier) 😞